_data/whats-new.yml

```diff
@@ -4,8 +4,64 @@ description: |
   We exclude from this list proofreading, spelling checks, and all minor updates.
 link: /whats-new.html
 thread: /whatsnew-feed.xml
-updated: Fri Aug 9 23:29:50 2019
+updated: Mon Aug 19 16:06:34 2019
 entries:
+- description: Added a tutorial about how to create a [grouped product using the REST
+    API](https://devdocs.magento.com/guides/v2.3/rest/tutorials/grouped-product/create-and-manage-grouped-products.html).
+  versions: 2.3.x
+  type: New topic
+  date: August 19, 2019
+  link: https://github.com/magento/devdocs/pull/5155`1  
+- description: Updated the [Set up multiple website or stores](https://devdocs.magento.com/guides/v2.3/cloud/project/project-multi-sites.html)
+    topic in the Cloud guide. Now, this topic differentiates between sites with a
+    shared domain and sites with unique domains.
+  versions: 2.x
+  type: Major update
+  date: August 15, 2019
+  link: https://github.com/magento/devdocs/pull/5123
+- description: Added the [Change increment ID for a database entity on particular
+    store](http://devdocs.magento.com/guides/v2.2/config-guide/multi-site/change-increment-id.html)
+    topic to the Configuration Guide.
+  versions: 2.2.x, 2.3.x
+  type: New topic
+  date: August 15, 2019
+  link: https://github.com/magento/devdocs/pull/5129
+- description: Added a code sample and updated the API reference in the [QuickSearch
+    Widget](https://devdocs.magento.com/guides/v2.3/javascript-dev-guide/widgets/widget_quickSearch.html)
+    topic.
+  versions: 2.2.x, 2.3.x
+  type: Major update
+  date: August 14, 2019
+  link: https://github.com/magento/devdocs/pull/5182
+- description: Provided details about what happens when you run the [Data Migration
+    Tool](https://devdocs.magento.com/guides/v2.2/migration/migration-migrate-data.html).
+  versions: 2.2.x, 2.3.x
+  type: Major update
+  date: August 14, 2019
+  link: https://github.com/magento/devdocs/pull/5167
+- description: Updated the Javascript resources topic with instructions for [including
+    third-party JavaScript libraries](https://devdocs.magento.com/guides/v2.3/javascript-dev-guide/javascript/js-resources.html#including-third-party-javascript-libraries).
+  versions: 2.2.x, 2.3.x
+  type: Major update
+  date: August 13, 2019
+  link: https://github.com/magento/devdocs/pull/5161
+- description: Added documentation for the [TrimInput widget](https://devdocs.magento.com//guides/v2.3/javascript-dev-guide/widgets/widget_trim-input.html)
```

Comment:
The link `https://devdocs.magento.com//guides` causes double slash failure when I generate a preview with rake.

Reply:
The link is redirected to home page. There is no such file `guides/v2.3/javascript-dev-guide/widgets/widget_trim-input.md`.

Reply:
@dshevtsov   Did you refresh?  I don't have the *.md file in the files that I generated.  Nor do I have a double slash.

Reply:
I didn't mean you have .md files.
The link points to the HTML page, but the page does not exist, because there is no corresponding .md file in the source.

Reply:
Thanks for clarifying.